prow/jobs/kyma-project/test-infra/images.yaml

```diff
@@ -91,7 +139,7 @@ postsubmits:
         prow.k8s.io/pubsub.runID: "pull-build-image-builder"
         prow.k8s.io/pubsub.topic: "prowjobs"
         preset-sa-kyma-push-images: "true"
-      run_if_changed: ^pkg/.*.go|cmd/image-builder/.*.go|^go.mod
+      run_if_changed: ^pkg/.*.go|cmd/image-builder/.*.go|^go.mod|cmd/image-builder/images/
```

Comment:
Do we want to rebuild it also when kaniko image has been changed?

Reply:
I would keep it that way, so we have always both versions of image-builder build for the same commit. I think there won't be much changes in dockerfiles and having images for the same commit will be easier to maintain.